howtohireme/juniorjobs
# 113: Using 'database_cleaner' gem to clean state testing

Issue #ID 113.

### Description

Using `database_cleaner` gem to make tests more accurate.

### Testing steps

* Step 1
Check that `juniorjobs-testing` database is clean and seeded. Size of testing database must be around development database.
* Step 2
Run `rspec`
* Step 3
Check if everything is green.
* Step 4
Check that `juniorjobs-testing` database size is remaining same.

### How to check if database is cleaned
* Step 1:
Login to psql:
`psql --host="127.0.0.1" --username='postgres' -W`
`--host` and `--username` may differ. Check your `database.yml` in `config` folder
* Step 2:
Enter password
* Step 3:
In psql console:
`\l+`
